guava/src/test/java/com/fasterxml/jackson/datatype/guava/MultiMapOrderMpEntriesByKeys7Test.java

```diff
@@ -0,0 +1,157 @@
+package com.fasterxml.jackson.datatype.guava;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.collect.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MultiMapOrderMpEntriesByKeys7Test extends ModuleTestBase {
```

Comment:
Could you add a comment linking to issue (I know number 7 suggests it, but explicitly)